apps/src/code-studio/pd/workshop_dashboard/reports/teacher_attendance_report.jsx

```diff
@@ -89,10 +89,6 @@ export default class TeacherAttendanceReport extends React.Component {
     return <a href={href} target="_blank" style={styles.link}>{workshop_id}</a>;
   };
 
-  formatUrl = (url) => {
```

Comment:
Was this just unused?

Reply:
Yeah, I just noticed that and remove it. It's (obviously) not related to this PR. I'd be happy to pull it out separately.